spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1199,77 +1156,72 @@ app register --name log --type sink --uri maven://org.springframework.cloud.stre
-
-Note that you can override these defaults by setting them as you would any
-application property value.
-
-Data Flow does not provide its own implementation to store and visualize
-historical metrics data. We integrate with existing ALM
-systems by providing an Exporter application that consumes messages from the
-same destination as the Metrics Collector and writes them to an existing ALM
-system. We have developed an Elastic Search exporter
-with a Grafana front end, because it is open source.
-
-=== Log and DataDog MetricWriter
-If you prefer to have deployed applications bypass the centralized collection
-of metrics through the Metrics Collector, you can use the MetricWriters in https://github.com/spring-cloud/spring-cloud-dataflow-metrics[Spring Cloud Data Flow
-Metrics] and https://github.com/spring-cloud/spring-cloud-dataflow-metrics-datadog[Spring Cloud Data Flow Metrics Datadog Metrics].
-
-The Data Flow Metrics project provides the foundation for exporting Spring Boot
-metrics through `MetricWriters`.  It provides Spring Boot's AutoConfiguration to set up
-the writing process and common functionality such as defining a metric name
-prefix appropriate for your environment.  For example, you may want to
-include the region where the application is running in addition to the
-application's name and stream/task to which it belongs.  It also includes a
-`LogMetricWriter`, so that metrics can be stored in a log file.  While
-simple in approach, log files are often ingested into application monitoring
-tools (such as Splunk), where they can be further processed to create dashboards
-of an application's performance.
-
-To make use of this functionality, you need to add additional dependencies
-into your Stream and Task applications.  To customize the "`out of the box`" Task
-and Stream applications, you can use the
-http://start-scs.cfapps.io/[Data Flow Initializr] to generate a project and
-then add to the generated Maven pom file the MetricWriter implementation you
-want to use.  The documentation on the Data Flow Metrics project pages provides
-the additional information you need to get started.
+spring.metrics.export.triggers.application.includes=integration**
+----
+
+For 2.x applications, the property is `spring.cloud.stream.metrics.meter-filter` and it does not have a default value, so all metrics are exported.
+
+Note that the Data Flow UI only displays only instantaneous input and output channel message rates.
+Data Flow does not provide its own implementation to store and visualize historical metrics data, instead we integrate with existing monitoring system.
+For Boot 1.x, there is support for sending metrics to the application log and DataDog.
+For Boot 2.x, metrics is backed by the https://micrometer.io/[Micrometer library] that provides a wide range of https://docs.spring.io/spring-boot/docs/2.0.0.RELEASE/reference/htmlsingle/#production-ready-metrics[monitoring systems].
+
+
+==== Spring Boot 2.x
```

Comment:
Maybe we can remove this for 1.5 M1. We can add it to 1.5 RC.

Reply:
Reworded to indicate it is a WIP.